var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -32,6 +32,7 @@ class Neuron(CMakePackage):
     variant("python",        default=True,  description="Enable python")
     variant("rx3d",          default=False,  description="Enable cython translated 3-d rxd")
     variant("tests",         default=False, description="Enable unit tests")
+    variant("caliper",         default=False, description="Add LLNL/Caliper support")
```

Comment:
@ikitayama : LGTM,  just few nitpicks  - could you indent `default` with the line on `34`, `when` clause of line `47` with `46`? Then its good to go!

Thank you!

Reply:
Done.